src/Alert.jl

```diff
@@ -0,0 +1,268 @@
+###################################################
+#
+# Copyright © Akamai Technologies. All rights reserved.
+# Proprietary and confidential.
+#
+# File: Alert.jl
+#
+# Functions to communicate with the mPulse Repository REST API regarding Alert Objects.
+# This file MUST be `include()`d from `mPulseAPI.jl`
+#
+###################################################
+
+
+
+export
+    deleteRepositoryAlert,
+    getRepositoryAlert,
+    postRepositoryAlert
+
+"""
+
+Fetches an Alert object from the mPulse repository
+
+The alert will be cached in memory for 1 hour, so subsequent calls using a matching `alertID` return
+quickly without calling out to the API.  This can be a problem if the alert changes in the repository.
+You can clear the cache for this tenant using [`mPulseAPI.clearAlertCache`](@ref) and passing in `alertID`.
+
+#### Arguments
+`token::AbstractString`
+:    The Repository authentication token fetched by calling [`getRepositoryToken`](@ref)
+
+#### Optional Arguments
```

Comment:
Change this to Keyword Arguments.  Also, you need to add `alertName`

Reply:
fixed

Reply:
were the fixes pushed?  I don't see them here.

Reply:
hm, it should be pushed? 
```
bos-mpkgb:DA_mPulseAPI amisingh$ git push origin DA_update_library 
Counting objects: 8, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (8/8), done.
Writing objects: 100% (8/8), 1.18 KiB | 1.18 MiB/s, done.
Total 8 (delta 6), reused 0 (delta 0)
remote: Resolving deltas: 100% (6/6), completed with 3 local objects.
To https://github.com/amsingh17/mPulseAPI.jl.git
   7628e8e..6f5caca  DA_update_library -> DA_update_library
```